### PR TITLE
Add semver 0.2.1

### PR DESCRIPTION
**Pull request by justinmchase**
⚡ The new cli isn't actually quite ready yet. Just getting this prepared.

# Description

This adds the binaries from the new `optum/semver-cli` to the homebrew tap.

## Testing

Once the repo is public I should be able to install via homebrew locally, after that I will publish this PR.

### CODE REVIEW

File: runiac.rb
Comment (justinmchase) [inline runiac.rb:1]: This file looks like an accident, part of some automation mishap perhaps? If I'm wrong I'll add it back.

Comment (robertcrockett) [reply runiac.rb:1]: @justinmchase - I think this file was a formula for runiac. I think it needs to remain intact.

Comment (justinmchase) [reply runiac.rb:1]: Ok, I'll just add it back but it does seem like this repo has two runiac formulas and the one deleted here appears to be a mistake to me.

Maybe it should be reviewed and removed in a different PR if that's true.

Screen Shot 2023-04-18 at 3 31 32 PM

**Review by robertcrockett — APPROVED** [review]
Thanks for the updates!

**Pull request merged**